SPECS/python-parallel-ssh/python-parallel-ssh.spec

```
@@ -0,0 +1,131 @@
+%global srcname parallel-ssh
+
+Name:           python-%{srcname}
+Version:        1.9.1
+Release:        10%{?dist}
+Summary:        Asynchronous parallel SSH library
+# https://github.com/ParallelSSH/parallel-ssh/pull/179
+License:        LGPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://parallel-ssh.readthedocs.io/
+Source0:        https://github.com/ParallelSSH/parallel-ssh/archive/refs/tags/%{version}.tar.gz#/%{srcname}-%{version}.tar.gz
+BuildRequires:  gcc
+
+%global _description \
+Library for running asynchronous parallel SSH commands over many hosts.\
+\
+parallel-ssh uses asychronous network requests - there is no multi-threading or\
+multi-processing used.\
+\
+This is a requirement for commands on many (hundreds/thousands/hundreds of\
+thousands) of hosts which would grind a system to a halt simply by having so\
+many processes/threads all wanting to execute if done with\
+multi-threading/processing.
+
+%description %{_description}
+
+%package -n python3-%{srcname}
+Summary:        %{summary}
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-Cython
+%{?python_provide:%python_provide python3-%{srcname}}
+
+%description -n python3-%{srcname} %{_description}
+
+Python 3 version.
+
+%prep
+%autosetup -n %{srcname}-%{version}
+rm -vr *.egg-info/
+find -type f -name '*.c' -print -delete
+
+%build
+%py3_build
+
+%install
+%py3_install
+
+%files -n python3-%{srcname}
+%doc README.rst
+%{python3_sitearch}/pssh/
+%{python3_sitearch}/parallel_ssh-*.egg-info/
+
+%changelog
+* Wed Jun 22 2022 Sumedh Sharma <sumsharma@microsoft.com> - 1.9.1-10
+- Initial CBL-Mariner import from fedora 35(License: LGPLv2)
```

This line should have the license of the spec file, not the license of the package.

```suggestion
- Initial CBL-Mariner import from Fedora 35 (License: MIT)
```

---

Done